app/utils/callback/lava.py

```diff
@@ -566,10 +554,10 @@ class LavaResults(object):
     ]
 
     def __init__(self, results, metadata):
-        self.groups = self._populate_groups(results,
```

Comment:
The commit message is empty, would be good to explain what this does to make the git history easier to read.